test/vm-build

```diff
@@ -31,20 +31,17 @@ args = parser.parse_args()
 
 def build(machine):
     """Build Cockpit in a virtual machine"""
-    srpm = subprocess.check_output([ "../tools/make-srpm" ]).strip()
+    source = subprocess.check_output([ "../tools/make-tar" ]).strip()
```

Comment:
Can we call this script 'make-source' to match the nomenclature you've used throughout all the other various and stuff. It also helps distinguish it from a real tarball (which is created via 'make dist').